torch/_inductor/kernel/conv.py

```diff
@@ -44,19 +45,33 @@ def conv_grid(n, c, h, w, meta):
     )
 
 
-conv_configs = functools.partial(
-    filtered_configs,
-    configs=(
-        # "BLOCK_M", "BLOCK_N", "BLOCK_K", "num_stages", "num_warps"
-        (64, 256, 16, 2, 4),
-        (256, 64, 16, 2, 4),
-        (1024, 16, 16, 1, 8),
-        (128, 128, 32, 2, 8),
-        (64, 64, 32, 2, 4),
-        (64, 256, 32, 2, 8),
-        (256, 64, 32, 2, 8),
-    ),
-)
+if torch.version.hip is None:
```

Comment:
@jataylo Please add a comment explaining why we are choosing different configs for ROCm.

Reply:
https://github.com/pytorch/pytorch/pull/107584/files#diff-3385fcbafbb6165d9e55060e0ddf92c46e17625eb6c0c38a07f32995950882daR63